src/openforms/authentication/contrib/digid_oidc/backends.py

```diff
@@ -0,0 +1,55 @@
+import logging
+
+from django.contrib.auth.models import AnonymousUser
+from django.core.exceptions import SuspiciousOperation
+from django.urls import reverse
+
+from mozilla_django_oidc_db.backends import (
+    OIDCAuthenticationBackend as _OIDCAuthenticationBackend,
+)
+
+from .constants import DIGID_OIDC_AUTH_SESSION_KEY
+from .mixins import SoloConfigMixin
+from .settings import OIDC_AUTHENTICATION_CALLBACK_URL
+
+logger = logging.getLogger(__name__)
+
+
+class OIDCAuthenticationDigiDBackend(SoloConfigMixin, _OIDCAuthenticationBackend):
+    """
+    Allows logging in via OIDC with DigiD
+    """
+
+    def authenticate(self, request, *args, **kwargs):
+        # Differentiate between DigiD authentication via OIDC and admin login
+        # via OIDC by checking the callback url
```

Comment:
Probably not needed if we call the backend directly in the view without relying on django's `auth.authenticate` functionality